src/displayapp/screens/Weather.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <cstdint>
+#include <lvgl/lvgl.h>
+#include "displayapp/screens/Screen.h"
+#include "components/ble/SimpleWeatherService.h"
+#include "displayapp/apps/Apps.h"
+#include "displayapp/Controllers.h"
+#include "Symbols.h"
+#include "utility/DirtyValue.h"
+
+namespace Pinetime {
+
+  namespace Controllers {
+    class Settings;
+  }
+
+  namespace Applications {
+    namespace Screens {
+
+      class Weather : public Screen {
+      public:
+        Weather(Controllers::Settings& settingsController, Controllers::SimpleWeatherService& weatherService);
+        ~Weather() override;
+
+        void Refresh() override;
+
+      private:
+        const char* getCondition(const Pinetime::Controllers::SimpleWeatherService::Icons icon);
```

Comment:
This function could be moved to displayapp/screens/WeatherSymbols.{h,cpp}, as I imagine it might be useful to other apps as well. Also, we use PascalCase for functions.